keycloak/keycloak-admin-ui
# Identity Providers(Mappers): Add create/edit functionality for mappers of type Username Template Importer

## Motivation

Towards #1206

## Verification Steps


1. Go to Identity Providers
2. Create a SAML provider
3. Go to the "mappers" tab
4. Create a new mapper
5. In the "Mapper Type" dropdown, select "Username Template Importer"
6. Verify that the correct fields in the form are shown -- you should see new form fields for "Target" and "Template".
7. Fill out the fields and save. 
8. Verify the save is successful.
9. Go back to mappers and select the new mapper you just created.
10. Edit the mapper, save, and verify the mapper has been updated successfully.

## Checklist:
